config/ember-try.js

```diff
@@ -1,84 +1,72 @@
 'use strict';
 
 const getChannelURL = require('ember-source-channel-url');
+const { embroiderSafe, embroiderOptimized } = require('@embroider/test-setup');
 
-module.exports = function () {
-  return Promise.all([getChannelURL('release'), getChannelURL('beta'), getChannelURL('canary')]).then((urls) => {
-    return {
```

Comment:
First off, sorry it's taken so long to review :( 

why drop 3.20?

Reply:
(because it's a breaking change if we drop it -- if tests pass with supporting 3.20, I don't see an issue with keeping it around)

Reply:
added a commit to add this back